builders/interoperability/xcm/xc20/send-xc20s/xcm-pallet.md

```diff
@@ -0,0 +1,364 @@
+
+    === "Polkadot.js API Example"
+        ```js
+        --8<-- 'code/builders/interoperability/xcm/xc20/send-xc20s/polkadot-xcm/check-xcm-version.js'
+        ```
+
+??? function "**palletVersion**() — returns current pallet version from storage"
+
+    === "Parameters"
+
+        None
+
+    === "Returns"
+
+        A number representing the current version of the pallet.
+
+        ```js
+        // If using Polkadot.js API and calling toJSON() on the unwrapped value
+        0
+        ```
+
+    === "Polkadot.js API Example"
+
+        ```js
+         --8<-- 'code/builders/interoperability/xcm/xc20/send-xc20s/polkadot-xcm/pallet-version.js'
+        ```
+
+### Pallet Constants {: #constants }
+
+There are no constants part of the PolkadotXCM pallet.
+
+## Building an XCM Message with the PolkadotXCM Pallet {: #build-with-PolkadotXCM-pallet}
+
+This guide covers the process of building an XCM message using the PolkadotXCM Pallet, specifically the `limitedReserveTransferAssets` function. It can be adapted for different assets using the other methods of the PolkadotXCM Pallet.
+
+!!! note
+    Each parachain can allow and forbid specific methods from a pallet. Consequently, developers must ensure that they use methods that are allowed, or the transaction will fail with an error similar to `system.CallFiltered`.
+
+You'll be transferring xcUNIT tokens, which are the [XC-20](/builders/interoperability/xcm/xc20/overview/){target=\_blank} representation of the Alphanet relay chain token, UNIT. You can adapt this guide for any other XC-20.
+
+### Checking Prerequisites {: #polkadotxcm-check-prerequisites}
+
+To follow along with the examples in this guide, you need to have the following:
+
+- An account with funds.
+ --8<-- 'text/_common/faucet/faucet-list-item.md'
+- Some xcUNIT tokens. You can swap DEV tokens (Moonbase Alpha's native token) for xcUNITs on [Moonbeam-Swap](https://moonbeam-swap.netlify.app/#/swap){target=\_blank}, a demo Uniswap-V2 clone on Moonbase Alpha
+
+    !!! note
+        You can adapt this guide to transfer another [external XC-20 or a local XC-20](/builders/interoperability/xcm/xc20/overview/){target=\_blank}. For external XC-20s, you'll need the asset ID and the number of decimals the asset has. For local XC-20s, you'll need the contract address.
+
+    ![Moonbeam Swap xcUNIT](/images/builders/interoperability/xcm/xc20/send-xc20s/xtokens-pallet/xtokens-1.webp)
+
+To check your xcUNIT balance, you can add the XC-20's [precompile address](/builders/interoperability/xcm/xc20/overview/#calculate-xc20-address){target=\_blank} to MetaMask with the following address:
+
+```text
+0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080
+```
+
+### PolkadotXCM Limited Reserve Transfer Assets Function {: #polkadotxcm-limited-reserve-transfer-assets-function}
```

Comment:
It should be `transferAssets` and `TransferAssetUsingTypeandThen`, the later allows you to combine reserve and non reserve assets. Try it out :)

Reply:
Fixed, thanks!